docs/embedded/development/content-experiences/search-content.md

```diff
@@ -507,9 +507,9 @@ Content-type: application/json
   ]
 }
 ```
-## Example 7: Search for content with specific content properties in response body
+## Example 7: Search for content with specific content properties in response body and sort the results
 
-This example queries container content by specific words and requires the response to include all specified attributes on the content.
+This example queries container content by specific words and requires the response to include all specified attributes on the content.  Properties that are [sortable](https://learn.microsoft.com/en-us/sharepoint/technical-reference/crawled-and-managed-properties-overview) can be used to sort the results.
```

Comment:
Per automated build report findings, invalid link:

- no absolute links, use relative
- no hard-coded locale in URL... remove